src/js/configurations/geography_sa.js

```diff
@@ -55,14 +62,49 @@ export class Config {
     }
 
     get defaultFilters() {
-        if (this.config["default_filters"] != undefined)
-            return this.config["default_filters"];
-        return [];
+        let globalConfig = [];
+        let viewConfig = [];
+        let mergedConfig = [];
+        if (
+            this.views[this._viewName] !== undefined
+            && this.views[this._viewName]["default_filters"] !== undefined
+        ) {
+            viewConfig = this.views[this._viewName]["default_filters"];
+        }
```

Comment:
This code block is repeated in defaultFilters and restrictValues. Do you think it's worth adding a function like so:

```
getViewConfig(filterType) {
    if (
        this.views[this._viewName] !== undefined
        && this.views[this._viewName][filterType] !== undefined
    ) {
        return this.views[this._viewName][filterType];
    }
    return null;
}
```

Then call from defaultFilters
```
viewConfig = getViewConfig("default_filters");
```

And the same for the global config block

Reply:
nice catch - done